opentelemetry-otlp/tests/integration_test/Cargo.toml

```diff
@@ -12,7 +12,8 @@ opentelemetry_sdk = { path = "../../../opentelemetry-sdk", features = ["rt-tokio
 opentelemetry-otlp = { path = "../../../opentelemetry-otlp", features = ["tonic", "metrics", "logs"] }
 opentelemetry-semantic-conventions = { path = "../../../opentelemetry-semantic-conventions" }
 opentelemetry-proto = { path = "../../../opentelemetry-proto", features = ["gen-tonic-messages", "trace", "with-serde"] }
+opentelemetry-appender-log = { path = "../../../opentelemetry-appender-log", default-features = false}
+log = { workspace = true }
 tokio = { version = "1.0", features = ["full"] }
 serde_json = "1"
 testcontainers = "0.15.0"
-# env_logger = "0.11.3" // uncomment if needed for local debugging.
```

Comment:
This won't be usable further for local debugging. As we can only have one subscriber with `log`.